src/js/components/settings/__snapshots__/plannotification.test.js.snap

```diff
@@ -16,7 +16,7 @@ Array [
       <div
         className=""
       >
-        Mender Open Source
+        Mender Starter
```

Comment:
can we use the const `plan.os` here?

Reply:
that's in a test snapshot - in these files is just the result of the rendering and they should never be touched by an editor... let alone include imports/ other js functionality